Explorer/Assets/DCL/Multiplayer/SDK/Systems/SceneWorld/WritePlayerTransformSystem.cs

```diff
@@ -0,0 +1,61 @@
+using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using CrdtEcsBridge.Components;
+using CrdtEcsBridge.Components.Transform;
+using CrdtEcsBridge.ECSToCRDTWriter;
+using DCL.Multiplayer.SDK.Components;
+using ECS.Abstract;
+using ECS.Groups;
+using ECS.LifeCycle.Components;
+using ECS.Prioritization.Components;
+using SceneRunner.Scene;
+using Utility;
+
+namespace DCL.Multiplayer.SDK.Systems.SceneWorld
+{
+    [UpdateInGroup(typeof(SyncedPreRenderingSystemGroup))]
+    // [UpdateBefore(typeof(CleanUpGroup))]
+    // [LogCategory(ReportCategory.PLAYER_AVATAR_BASE)]
+    public partial class WritePlayerTransformSystem : BaseUnityLoopSystem
+    {
+        private readonly IECSToCRDTWriter ecsToCRDTWriter;
+        private readonly ISceneData sceneData;
+
+        public WritePlayerTransformSystem(World world, IECSToCRDTWriter ecsToCRDTWriter, ISceneData sceneData) : base(world)
+        {
+            this.ecsToCRDTWriter = ecsToCRDTWriter;
+            this.sceneData = sceneData;
+        }
+
+        protected override void Update(float t)
+        {
+            HandleComponentRemovalQuery(World);
+            UpdateSDKTransformQuery(World);
+        }
+
+        [Query]
+        [None(typeof(DeleteEntityIntention))]
+        private void UpdateSDKTransform(ref PlayerCRDTEntity playerCRDTEntity, ref SDKTransform sdkTransform)
```

Comment:
In the scene context `PlayerCRDTEntity` is changed to `PlayerSceneCRDTEntity`

Reply:
Tackled at https://github.com/decentraland/unity-explorer/pull/1030/commits/b6c7fd5847495e03c06bf63e3a9c6c3536e72fd8